vips/arithmetic.go

```diff
@@ -99,3 +100,16 @@ func vipsFindTrim(in *C.VipsImage, threshold float64, backgroundColor *Color) (i
 
 	return int(left), int(top), int(width), int(height), nil
 }
+
+// https://libvips.github.io/libvips/API/current/libvips-arithmetic.html#vips-getpoint
+func vipsGetPoint(in *C.VipsImage, n int, x int, y int) ([]float64, error) {
```

Comment:
What I don't understand is why `n` is being passed in here. Isn't that an output value from vips to designate the length of the output vector? I don't think that needs to be referenced from Go, and only used to create the Go array of the appropriate size.